Makefile

```diff
@@ -1,6 +1,26 @@
 package=github.com/Cox-Automotive/terraform-provider-alks
 GOFMT_FILES?=$$(find . -name '*.go' | grep -v vendor)
 
+format:
+	gofmt -w $(GOFMT_FILES)
+
+build:
+	go fmt
+	go build -v -o examples/terraform-provider-alks -mod=vendor .
+
+test:
+	go test -v .
+
+plan:
+	@terraform plan
+
+install:
+	go get -t -v ./...
+
+release:
+package=github.com/Cox-Automotive/terraform-provider-alks
+GOFMT_FILES?=$$(find . -name '*.go' | grep -v vendor)
+
```

Comment:
Your commit duplicated these lines.

Reply:
Good catch - hadn't seen that last night